BParkingNano/plugins/SimpleFlatTableProducer.cc

```diff
@@ -0,0 +1,246 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/Common/interface/ValueMap.h"
+#include "DataFormats/NanoAOD/interface/FlatTable.h"
+
+#include "CommonTools/Utils/interface/StringCutObjectSelector.h"
+#include "CommonTools/Utils/interface/StringObjectFunction.h"
+
+#include <vector>
+#include <boost/ptr_container/ptr_vector.hpp>
+
+template<typename T, typename TProd>
+class SimpleFlatTableProducerBase : public edm::stream::EDProducer<> {
```

Comment:
Is this needed? In ```CMSSW_10_2_15_patch2``` the template is defined in ```PhysicsTools/NanoAOD/interface/``` so it should be easy to include that file and only define the needed specializations

Reply:
thanks correct!